bin/www

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env node
```

Comment:
Check out the comments below. I think you can probably delete this file and just use `node app.js` to start the server with a few changes.